Druid/DRPiAnyMaskConstraint.class.st

```diff
@@ -6,6 +6,14 @@ Class {
 	#tag : 'IR-Paths'
 }
 
+{ #category : 'arithmetic' }
+DRPiAnyMaskConstraint >> asZ3ConstraintOnVar: aZ3Variable [
+
+	^ (aZ3Variable bitAnd:
+		  (self ifConstantDo: [ :val | val ] ifNot: [ :val | val name ]))
+		  ~== 0
+]
+
```

Comment:
Strange to see z3 stuff creeping into this class

Reply:
Ok, reading the code below I see that all constraints implement `asZ3ConstraintOnVar:` right?